proxy/ParentSelection.cc

```diff
@@ -1026,6 +1154,21 @@ EXCLUSIVE_REGRESSION_TEST(PARENTSELECTION)(RegressionTest * /* t ATS_UNUSED */,
     params = new ParentConfigParams(ParentTable);                                                                          \
   } while (0)
 
+// set's the filename with a recognized yaml suffix for testing a yaml config.
+#define REBUILD_YAML                                                                                                       \
```

Comment:
Why `#define` instead of an inline function or lambda?

Reply:
@SolidWallOfCode This is only used for unit tests, see also REBUILD for the current unit-tests.  Do you think I should change this?

Reply:
Yes. Test code should be held to the same coding standards as regular code.

Reply:
There are 212 tests using these test macros, T, REBUILD, REBUILD_YAML, REINIT, ST, RE, and FP.  Would it be acceptable to create an issue to circle back and refactor the unit-tests in a separate PR?